### PR TITLE
Add`FailureIssueThreshold` & `RecoveryThreshold` to MonitorConfig

### PR DESCRIPTION
**Pull request by cleptric**
https://github.com/getsentry/team-sdks/issues/43

https://develop.sentry.dev/sdk/check-ins/

### CODE REVIEW

**Review by antonpirker — APPROVED** [review]
lgtm

**Comment by antonpirker** [comment]
Maybe add a test where those parameters are omitted?

File: check_in.go [hunk 1]
```diff
@@ -87,6 +87,10 @@ type MonitorConfig struct { //nolint: maligned // prefer readability over optima
 	// A tz database string representing the timezone which the monitor's execution schedule is in.
 	// See: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 	Timezone string `json:"timezone,omitempty"`
+	// The number of consecutive failed check-ins it takes before an issue is created.
+	FailureIssueThreshold int64 `json:"failure_issue_threshold,omitempty"`
+	// The number of consecutive OK check-ins it takes before an issue is resolved.
+	RecoveryThreshold int64 `json:"recovery_threshold,omitempty"`
```
Comment (aldy505) [inline check_in.go:93]: Question: What happen if the users set a negative number?

Comment (cleptric) [reply check_in.go:93]: We can make it a unit 🙂

**Pull request merged**